bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/handler/NanoleafControllerHandler.java

```diff
@@ -309,10 +317,40 @@ private synchronized void stopPanelDiscoveryJob() {
         }
     }
 
+    private synchronized void startTouchJob() {
+        NanoleafControllerConfig config = getConfigAs(NanoleafControllerConfig.class);
+        if (config.deviceType != DEVICE_TYPE_CANVAS) {
+            logger.debug("NOT starting TouchJob for Panel {} because it has wrong device type {}",this.getThing().getUID(),config.deviceType);
+            return;
+        }
+        else
+            logger.debug("Starting TouchJob for Panel {}",this.getThing().getUID());
+
+
+
+        if (StringUtils.isNotEmpty(getAuthToken())) {
+            if ((touchJob == null || touchJob.isCancelled())) {
+                logger.debug("Starting Touchjob now");
+                touchJob = scheduler.schedule(this::runTouchDetection, 0, TimeUnit.SECONDS);
+            }
+        } else {
+            logger.error("starting TouchJob for Controller {} failed - missing token",this.getThing().getUID());

```

Comment:
In [E.4 of the coding guidelines](http://docs.openhab.org/developers/development/guidelines.html#e-logging) we listed what kind of logging level should be used when. Error should only be used:

> `error` logging should only be used to inform the user that something is tremendously wrong in his setup, the system cannot function normally anymore, and there is a need for immediate action. 

Please change this to use the correct logging levels and if possible change the ONLINE/OFFLINE status of thing / bridge.